kidney_transplant_llm/pydantic_study_variables.py

```diff
@@ -30,8 +30,8 @@ class DonorTypeMention(SpanAugmentedMention):
     )
 
 class DonorRelationship(StrEnum):
-    RELATED = 'Donor was related to the renal transplant recipient'
-    UNRELATED = 'Donor was unrelated to the renal transplant recipient'
+    RELATED = 'Donor was biologically related to the renal transplant recipient'
```

Comment:
Most of the time the word "biologically related" doesn't exist, it just says "living related donor" or "living donor from father" and no specification about biological relation.

Reply:
Prompt needs to account for "step-father" outlier case

Reply:
In general I think we should ASSUME if stated "father", "mother", "family donor", etc, that means biologically related **unless specified (step-father, step-mother)**

Reply:
OK lets keep it "as-is"

Reply:
OK we can add the word "biologically"